orderer/common/cluster/util.go

```diff
@@ -32,8 +37,13 @@ import (
 	"github.com/hyperledger/fabric/protoutil"
 	"github.com/pkg/errors"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/peer"
 )
 
+//
+const KeyingMaterialLabel = "EXPERIMENTAL label for orderer"
```

Comment:
can we change it to something not experimental? like - "orderer v3 authentication label" ?